lib/utils/prettier-utils.js

```diff
@@ -0,0 +1,72 @@
+const crossSpawn = require('cross-spawn');
+const logger = require('@blackbaud/skyux-logger');
+const fs = require('fs-extra');
+const latestVersion = require('latest-version');
+const path = require('path');
+
+const jsonUtils = require('./json-utils');
+
+/**
+ * Adds Prettier configuration files to the project.
+ */
+ async function configurePrettier(projectPath) {
+  await fs.ensureDir(path.join(projectPath, '.vscode'));
+
+  await jsonUtils.writeJson(
+    path.join(projectPath, '.vscode', 'settings.json'),
+    {
+      'editor.defaultFormatter': 'esbenp.prettier-vscode',
+      'editor.formatOnSave': true,
+      'prettier.requireConfig': true
+    }
+  );
+
+  await jsonUtils.writeJson(
+    path.join(projectPath, '.prettierrc'),
+    {
+      singleQuote: true
+    }
+  );
+}
+
+/**
+ * Runs Prettier on source files.
+ */
+ function applyPrettierToFiles(projectPath) {
+  logger.info('Applying Prettier to project...');
+
+  crossSpawn.sync(
+    'npx',
+    [
+      'prettier',
+      '--write',
+      '.'
+    ],
+    {
+      stdio: 'inherit',
+      cwd: projectPath
+    }
+  );
+
+  logger.info('Done.');
+}
+
+async function addPrettierToDevDependencies(packageJson) {
+  packageJson.devDependencies = packageJson.devDependencies || {};
+
+  const packages = await Promise.all(
+    [
+      latestVersion('prettier'),
+      latestVersion('eslint-config-prettier')
+    ]
+  );
+
+  packageJson.devDependencies['prettier'] = packages[0];
+  packageJson.devDependencies['eslint-config-prettier'] = packages[1];
```

Comment:
We're also installing `eslint-plugin-prettier` on our builders. Should we install that with our consumers, too? 
See: https://github.com/prettier/eslint-plugin-prettier#recommended-configuration

Reply:
When I was doing research on how to integrate Prettier and ESLint, I came across this guidance from Prettier which seemed to discourage the use of `eslint-plugin-prettier`. I think `eslint-config-prettier` will give us what we need without the plugin. https://prettier.io/docs/en/integrating-with-linters.html#notes

Reply:
Good info. Thanks!